tools/api-markdown-documenter/src/api-item-transforms/TsdocNodeTransforms.ts

```diff
@@ -114,6 +116,15 @@ export function _transformTsdocNode(
 		case DocNodeKind.HtmlEndTag: {
 			return transformTsdocHtmlTag(node as DocHtmlEndTag, options);
 		}
+		case DocNodeKind.InheritDocTag: {
```

Comment:
We haven't hit this case to my knowledge, but if we do, a more tailored error message seemed appropriate.